front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/ObjectType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.model;
+
+import com.netflix.spinnaker.front50.model.application.Application;
+import com.netflix.spinnaker.front50.model.notification.Notification;
+import com.netflix.spinnaker.front50.model.pipeline.Pipeline;
+import com.netflix.spinnaker.front50.model.project.Project;
+import com.netflix.spinnaker.front50.model.serviceaccount.ServiceAccount;
+import com.netflix.spinnaker.front50.model.snapshot.Snapshot;
+import com.netflix.spinnaker.front50.model.tag.EntityTags;
+
+public enum ObjectType {
+  PROJECT(Project.class, "projects", "project-metadata.json"),
+  PIPELINE(Pipeline.class, "pipelines", "pipeline-metadata.json"),
+  STRATEGY(Pipeline.class, "pipeline-strategies", "pipeline-strategy-metadata.json"),
+  NOTIFICATION(Notification.class, "notifications", "notification-metadata.json"),
+  SERVICE_ACCOUNT(ServiceAccount.class, "serviceAccounts", "serviceAccount-metadata.json"),
```

Comment:
`SERVICE_ACCOUNT`, `APPLICATION_PERMISSION` and `SNAPSHOT` do not fit the typical naming conventions for metadata.

I have no idea if anyone on aws is using these otherwise I'd be inclined to make them consistently named (metadata filename in particular) ... nobody at Netflix is using them.